compose/compiler/compiler-hosted/src/main/java/androidx/compose/compiler/plugins/kotlin/lower/ComposableFunctionBodyTransformer.kt

```diff
@@ -3241,16 +3248,18 @@ class ComposableFunctionBodyTransformer(
             if (
                 // if no wrapping group but some results have calls, we have to have every result
```

Comment:
can you update this comment from "some results have calls" to "more than one result has calls" ?